Dockerfile

```diff
@@ -1,4 +1,9 @@
-FROM folioci/alpine-jre-openjdk11:latest
+FROM folioci/alpine-jre-openjdk17:latest
+
+# Install latest patch versions of packages: https://pythonspeed.com/articles/security-updates-in-docker/
+USER root
+RUN apk upgrade --no-cache
+USER folio
```

Comment:
Is this part of the documented steps for upgrading to JDK 17 or something additional included in this work?

Reply:
yes @marcjohnson-kint PFB the link of the same which I followed for the **_java 17 upgradation documentation_**

Link: https://wiki.folio.org/display/TC/JDK+17+and+Java+17

![image](https://github.com/folio-org/mod-users/assets/104345088/0b551484-dd33-44ca-9f18-23b63642b3bf)

Reply:
@nirmalr-shah Thanks for confirming, it seems odd to me that's included in these upgrade instructions.

I'll resolve this